src/argilla/client/feedback/training/schemas.py

```diff
@@ -128,30 +141,30 @@ def for_text_classification(
             RankingQuestionUnification,
         ],
         label_strategy: str = None,
-    ) -> "TrainingTaskMappingForTextClassification":
+    ) -> "TrainingTaskForTextClassification":
         """
         _summary_
 
         Args:
             text (TextField): The TextField to use for training.
-            label (Union[RatingQuestion, LabelQuestion, MultiLabelQuestion, RatingQuestionUnification, LabelQuestionUnification, MultiLabelQuestionUnification]): _description_
+            label (Union[RatingQuestion, LabelQuestion, RankingQuestion, MultiLabelQuestion, RatingQuestionUnification, LabelQuestionUnification, MultiLabelQuestionUnification, RankingQuestionUnification]): _description_
```

Comment:
As mentioned before, I think we can just remove the type-hints from the docstrings are those are automatically included when generating the docs if the type-hint is defined in the function arguments

Reply:
Note that this comment also applies to the rest of the methods defined in this Python file

Reply:
Sure, we can do that

Reply:
1eff2089fbcf0ff6e21848da0781e600acf407fb